conftest.py

```diff
@@ -4,7 +4,7 @@
 
 from syrupy.utils import env_context
 
-typing.TYPE_CHECKING = True
+typing.TYPE_CHECKING = False
```

Comment:
Mypy already validates type checking and `typing.Protocol` not available python <3.8. `Protocol` is used for defining types which is only used in development.